default/props.conf

```diff
@@ -1,25 +1,41 @@
-
 [postfix_syslog]
 # Note:  Extending Splunk default settings (See $SPLUNK_HOME/etc/system/default/props.conf)
-EXTRACT-queue_id    = postfix/\w+\[\d+\]:\s+(?<queue_id>[A-Fa-f0-9]{6,20}):
+
+# Extract the subject if enabled in Postfix's configuration
 REPORT-subject = postfix_subject
-# Extract to/from/message-id (without the '<>'s)
+# Extract to/from/message-id/helo (without the '<>'s)
 REPORT-angle_brackets = postfix_angle_brackets
-EXTRACT-reason = \((?<reason>[^)]*)\)
-EXTRACT-reject_reason = >: (?<reject_reason>[^;]*);
+
+EXTRACT-queue_id = postfix/\w+\[\d+\]:\s+(?<queue_id>[A-Fa-f0-9]{6,20}):
+EXTRACT-bounce = postfix/bounce\[\d+\]: [a-fA-F0-9]{6,20}: (?<bounce_reason>[^:]+): (?<bounce_queue_id>[a-fA-F0-9]{6,20})$
+EXTRACT-status_reject = postfix/smtpd\[\d+\]: (?:NOQUEUE|[A-Fa-f0-9]{6,20}): (?<status>reject):
+EXTRACT-reason = status=[^\s]+\s+\((?<reason>.*)\)$
+EXTRACT-reject_reason = : (?<reject_reason>[^;:]+);
 EXTRACT-dest = relay=(?<dest_host>[^\[ ,]+)\[(?<dest_ip>[^: \]]+)\](?::(?<dest_port>\d+))?
 EXTRACT-remote_queue = queued as (?<xref>[A-Fa-f0-9]+) in reason
-EXTRACT-status_code = status=\w+ \((?:host \S+ said:\s*)?(?<status_code>\d+)
+#EXTRACT-status_code = status=\w+ \((?:host \S+ said:\s*)?(?<status_code>\d+)
 EXTRACT-src-connect = (?:dis)?connect(?:ion after (?:HELO|CONNECT))? from (?:(?<src_host>[^\[]+)\[(?<src_ip>[\d.]+)|(?<src>.*))
+# Extration of the different delays (cf. http://logreporters.sourceforge.net/faq.html#percentiles)
+EXTRACT-delays = ^(?<time_before_queue>[^/]+)/(?<time_in_queue>[^/]+)/(?<time_connecting>[^/]+)/(?<time_transmitting>[^$]+)$ in delays
 
 # Rename fields for CIM compliance with the Email data model
-FIELDALIAS-queue = queue_id AS internal_message_id
-FIELDALIAS-process = pid AS process_id
-FIELDALIAS-tofrom = to AS recipient, from AS src_user
-FIELDALIAS-nrcpt = nrcpt as recipient_count
-# Don't extract 'src_host' if "unknown" (typical with reverse DNS disabled)
-EVAL-src = coalesece(src, nullif(src_host,"unknown"), src_ip)
-# FIELDALIAS-src = src_host AS src, src_ip AS src
+FIELDALIAS-status_code = dsn as status_code
```

Comment:
Names of these really don't matter.   I've not really seen any single convention emerge and win out based on  the dozens of TAs that I've looked over.

That said, this is clean and straight forward.  works for me